docs/tutorial/part-zero/index.md

```diff
@@ -1,194 +1,194 @@
 
-1.  Open up your terminal.
-2.  Run `gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world`. (_Note: Depending on your download speed, the amount of time this takes will vary. For brevity's sake, the gif below was paused during part of the install_).
-3.  Run `cd hello-world`.
-4.  Run `gatsby develop`.
+1. Buka terminal Anda.
+2. Jalankan `gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world`. (_Catatan: Tergantung pada kecepatan unduhan Anda, jumlah waktu yang dibutuhkan akan bervariasi. Untuk mempersingkat, gif di bawah ini ditunda selama proses install_).
+3. Jalankan `cd hello-world`.
+4. Jalankan `gatsby develop`.
 
 <video controls="controls" autoplay="true" loop="true">
   <source type="video/mp4" src="./03-create-site.mp4" />
   <p>Sorry! You browser doesn't support this video.</p>
 </video>
 
-What just happened?
+Apa yang baru saja terjadi?
 
 ```shell
 gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world
 ```
 
-- `new` is a gatsby command to create a new Gatsby project.
-- Here, `hello-world` is an arbitrary title — you could pick anything. The CLI tool will place the code for your new site in a new folder called “hello-world”.
-- Lastly, the GitHub URL specified points to a code repository that holds the starter code you want to use.
+- `new` adalah perintah gatsby untuk membuat proyek Gatsby yang baru.
+- Disini, `hello-world` adalah judul acak - Anda bisa memilih apa saja. CLI akan menempatkan kode untuk situs baru Anda pada folder baru yang disebut “hello-world”.
+- Terakhir, GitHub URL akan menentukan poin ke repositori yang berisi kode starter yang akan Anda gunakan.
 
 ```shell
 cd hello-world
 ```
 
-- This says 'I want to change directories (`cd`) to the “hello-world” subfolder'. Whenever you want to run any commands for your site, you need to be in the context for that site (aka, your terminal needs to be pointed at the directory where your site code lives).
+- Ini dapat diartikan juga sebagai 'Saya ingin mengubah direktori (`cd`) menjadi *subfolder* “hello-world”'. Kapan pun Anda ingin menjalankan perintah untuk situs Anda, Anda harus berada dalam direktori situs tersebut (alias, terminal Anda perlu diarahkan ke direktori tempat kode situs Anda berada).
 
 ```shell
 gatsby develop
 ```
 
-- This command starts a development server. You will be able to see and interact with your new site in a development environment — local (on your computer, not published to the internet).
+- Perintah ini akan memulai *server* pengembangan. Anda akan dapat melihat dan berinteraksi dengan situs baru Anda di lingkungan pengembangan - lokal (di komputer Anda, tidak dipublikasikan ke internet).
```

Comment:
Change it with "pengembangan server". Also, server already equivalent in Bahasa. Please find it here for reference: [https://kbbi.kemdikbud.go.id/entri/server](https://kbbi.kemdikbud.go.id/entri/server)